src/content/docs/apm/agents/php-agent/other-integrations/magento-business-insights.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Magento business insights
+tags:
+    - New Relic integrations
+    - Magento
+metaDescription: Monitor your Magento application's performance with our comprehensive Magento business insights dashboard. Track key metrics like page load time, server response time, transaction time, error rate, and availability.
+---
+
+import apmMagentoDashboardOverview from 'images/apm_screenshot-full_magento-dashboard-overview.png'
+
+Our Magento business insights dashboard provides a comprehensive view of your Magento ecommerce application's performance. With the dashboard, you can easily monitor key metrics like page load time, server response time, transaction time, error rate, and availability. You can also use the dashboard to diagnose issues in your app's performance and optimize your code.
+
+<img
+    title="Magento dashboard overview"
+    alt="A screenshot of a dashboard with a variety of graphs and charts showing data about a Magento ecommerce application."
+    src={apmMagentoDashboardOverview}
+/>
+
+<figcaption>
+    Our Magento business insights dashboard gives you a central view of your Magento app data. 
+</figcaption>
+
+To start monitoring your Magento app in New Relic, follow these steps to send your Magento app data to New Relic and view it in the Magento business insights dashboard.
+
+<Steps>
+    <Step>
```

Comment:
is the infra agent optional? In most of the recent quickstart dashboard integrations, it's optional so just checking if that's true for this one